app/src/main/java/xyz/whereuat/whereuat/KeyLocDialogFragment.java

```diff
@@ -79,6 +86,18 @@ public void onFocusChange(View v, boolean hasFocus) {
     }
 
     private void addKeyLoc(Context context, String name, Location loc) {
-        new KeyLocation(name, loc.getLatitude(), loc.getLongitude()).dbInsert(context);
+        double latitude = loc.getLatitude();
```

Comment:
I would rather take the line length penalty than create two more variables that are only used once, especially since this function signature is pretty short and this will still be pretty readable.